chart/crds/operator.istio.io_istiocnis.yaml

```diff
@@ -1039,6 +1039,10 @@ spec:
                           enabled:
                             description: Controls whether ambient redirection is enabled
                             type: boolean
+                          ipv6:
```

Comment:
hmm. as we don't support ambient yet, maybe we want to remove this for now?

Reply:
Removed. Thanks.